src/renderer/components/uielements/chart/chart.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { useMemo } from 'react'
+
+import themes, { ThemeType } from '@thorchain/asgardex-theme'
+import { Grid } from 'antd'
+import moment from 'moment'
+import { useObservableState } from 'observable-hooks'
+import { defaults } from 'react-chartjs-2'
+
+import { useThemeContext } from '../../../contexts/ThemeContext'
+import { abbreviateNumber } from '../../../helpers/numberHelper'
+import { LoadingView } from '../../shared/loading'
+import {
+  ChartContainer,
+  HeaderContainer,
+  TypeContainer,
+  TimeContainer,
+  HeaderToggle,
+  ChartWrapper,
+  BarChart,
+  LineChart
+} from './chart.style'
+import { ChartData, ChartTimeFrame } from './types'
+import { getDisplayData } from './utils'
+
+type Props = {
+  chartData: ChartData
+  chartIndexes: string[]
+  selectedIndex: string
+  selectChart: (value: string) => void
+}
+
+// https://www.chartjs.org/docs/latest/general/fonts.html#missing-fonts
+defaults.global.defaultFontSize = 14
+defaults.global.defaultFontStyle = 'normal'
+
+const DefaultChart: React.FC<Props> = React.memo(
+  (props: Props): JSX.Element => {
+    const { chartIndexes = [], chartData, selectedIndex, selectChart } = props
+    const [chartTimeframe, setChartTimeframe] = React.useState<ChartTimeFrame>('allTime')
+
+    const isDesktopView = Grid.useBreakpoint()?.md ?? false
+
+    const { themeType$ } = useThemeContext()
+    const themeType = useObservableState(themeType$)
```

Comment:
To avoid `undefined` as initial state, add always a default value to `useObservableState`:
```ts
const themeType = useObservableState(themeType$, ThemeType.LIGHT)
```